impeller/entity/contents/content_context.cc

```diff
@@ -405,13 +405,12 @@ bool ContentContext::IsValid() const {
   return is_valid_;
 }
 
-std::shared_ptr<Texture> ContentContext::MakeSubpass(
+fml::StatusOr<RenderTarget> ContentContext::MakeSubpass(
```

Comment:
I see, this is to make just reusing the same RenderTarget easy in the blur. I think this is fine.

Reply:
Yea, I had to do RenderTarget instead of Texture because of MSAA.